api/src/main/java/org/openmrs/Concept.java

```diff
@@ -1556,6 +1556,25 @@ public void setId(Integer id) {
 		}
 		return Collections.unmodifiableList(conceptMembers);
 	}
+
+	/**
+	 * If <code>includeRetired</code> is true, then the returned object is the actual stored list of
```

Comment:
Can please clean up the javadocs? The second phrase is probably unnecessary, the javadocs need to mention what the method does, i.e. returns set members and includes retired ones if includeRetired is set to true.